resource/src/routes/route.mustache

```diff
@@ -0,0 +1,57 @@
+const { Router } = require('express');
+const {{camelCasePlural}} = require('../controllers/{{snakeCasePlural}}');
+
+const serialize = ({{camelCase}}) => {
+  const data = Object.assign({}, {{camelCase}});
+  {{#isUser}}
+  delete(data.password);
+  {{/isUser}}
+  return data;
+};
+
+const router = module.exports = new Router()
+  .get('', function * (req, res) {
+    const list = yield {{camelCasePlural}}.all(req.query);
+    res.json(list.map(serialize));
+  })
+  .get('/:id', function * (req, res) {
+    const {{camelCase}} = yield {{camelCasePlural}}.find(req.params.id);
+    res.json(serialize({{camelCase}}));
+  })
+  .post('', function * (req, res) {
+    const {{camelCase}} = yield {{camelCasePlural}}.create(req.body);
+    res.status(201).json(serialize({{camelCase}}));
+  })
+  .put('/:id', function * (req, res) {
+    const {{camelCase}} = yield {{camelCasePlural}}.update(req.params.id, req.body);
```

Comment:
PUT vs. PATCH. you're probably right. many teams use PUT as a PATCH, and it's okay. if you insist this is trivial to change.

as for `super`, i don't see anything wrong with it really. it just helps reusing the base behavior. otherwise you will have to either dup implementations and shoot yourself in the foot when one of them changes, or use a mess of `old_method.apply(this, arguments)`. prototypes delegation is a thing in javascript, whether you like it or not it's used literally everywhere and the reason for it is that it's useful